src/pages/graphql/schema/b2b/purchase-order/mutations/add-items-to-cart.md

```diff
@@ -0,0 +1,73 @@
+---
+title: addPurchaseOrderItemsToCart mutation | Commerce Web APIs
+edition: b2b
+---
+
+# addPurchaseOrderItemsToCart mutation
+
+The `addPurchaseOrderItemsToCart` mutation adds all purchase order items to the shopping cart. The request must specify a `cart_id` and a `purchase_order_uid`. If the value of `replace_existing_cart_items` is set `true`, the mutation deletes the pre-existing items from the purchase order. Otherwise, the mutation merges the pre-existing items with the new items.
```

Comment:
```suggestion
The `addPurchaseOrderItemsToCart` mutation adds all purchase order items to the shopping cart. The request must specify a `cart_id` and a `purchase_order_uid`. If the value of `replace_existing_cart_items` is set to `true`, the mutation deletes the pre-existing items from the purchase order. Otherwise, the mutation merges the pre-existing items with the new items.
```